src/Poly.jl

```diff
@@ -49,6 +49,8 @@ number_of_variables(a::PolyRing) = 1
 
 characteristic(a::PolyRing) = characteristic(base_ring(a))
 
+is_finite(a::PolyRing) = false
```

Comment:
this is wrong in the case of the base ring `residue_ring(ZZ, 1)[1]` (aka the zero ring)